processing file asmFiles/cc1.asm (1 of 8)
processing file asmFiles/cc2.asm (2 of 8)
--- memsim.hex	2021-12-05 16:04:56.481365911 -0500
+++ memcpu.hex	2021-12-05 16:04:56.429364049 -0500
@@ -10,6 +10,6 @@
 :04008300AC4300008A
 :04008400AC43000089
 :04008500FFFFFFFF7B
-:0407F0000000BEF354
-:040BF0000000C00B36
+:0407F0000000BEEF58
+:040BF0000000C0073A
 :00000001FF
processing file asmFiles/cc3.asm (3 of 8)
--- memsim.hex	2021-12-05 16:04:58.887452107 -0500
+++ memcpu.hex	2021-12-05 16:04:58.807449241 -0500
@@ -6,6 +6,4 @@
 :0400810020630004F4
 :04008200AC4300008B
 :04008300FFFFFFFF7D
-:0407F0000000000401
-:040BF00000000004FD
 :00000001FF
processing file asmFiles/dual.llsc.asm (4 of 8)
--- memsim.hex	2021-12-05 16:05:01.872559045 -0500
+++ memcpu.hex	2021-12-05 16:05:01.788556035 -0500
@@ -13,8 +13,6 @@
 :04000C00E0430008C5
 :04000D00AC430008F8
 :04000E00FFFFFFFFF2
-:0400200000000002DA
-:0400240000000002D6
 :0400800034011FC068
 :0400810034020090B5
 :040082008C430000AB
processing file asmFiles/example.asm (5 of 8)
--- memsim.hex	2021-12-05 16:05:04.578655987 -0500
+++ memcpu.hex	2021-12-05 16:05:04.470652118 -0500
@@ -38,7 +38,4 @@
 :04008D008FBF000021
 :04008E0027BD000486
 :04008F0003E0000882
-:040090000000000369
-:040FFE0000000008E7
-:041FFE0000000208D5
 :00000001FF
processing file asmFiles/palgorithm.asm (6 of 8)
